cmd/notation/internal/truststore/truststore.go

```diff
@@ -83,10 +89,13 @@ func ListCerts(root string, depth int) error {
 			return err
 		}
 		if info.Mode().IsRegular() {
-			if _, err := corex509.ReadCertificateFile(path); err != nil {
+			certs, err := corex509.ReadCertificateFile(path)
+			if err != nil {
 				return err
 			}
-			fmt.Println(path)
+			if len(certs) != 0 {
+				fmt.Println(path)
+			}
```

Comment:
Why do we need to display path?

Reply:
Oh, this is the result display of the `notation cert list` command.